### ISSUE TITLE
Thin white line on hold note bodies

### ISSUE THREAD

**Issue by enoslayd** [ppy/osu#10738]
**Describe the bug:** theres a very thin semi transparent white line on the side of hold note bodies. in the screenshit, it appears on red hold note. it doesnt happen on stable with the skin in screenshot. on classic skin, its not very noticeable except on the right side of the yellow hold note.

**Screenshots or videos showing encountered issue:** 
Lazer:
![osu_2020-11-08_15-03-29](https://user-images.githubusercontent.com/41866328/98459416-9ebeff00-21d5-11eb-8d29-78a4d9893f0e.png)
classic skin:
![osu_2020-11-08_15-15-52](https://user-images.githubusercontent.com/41866328/98459462-ec3b6c00-21d5-11eb-9b7f-2a1a09852308.png)

Stable:
![screenshot139](https://user-images.githubusercontent.com/41866328/98459454-dd54b980-21d5-11eb-9fc7-a05a7c4ceb41.png)

**skin:** https://github.com/enoslayd/outline-skin (the outline folder is the skin)

**osu!lazer version:** master after #10724  merge

**Comment by peppy** [comment]
Stable fixes incorrect transparent values in textures, lazer doesn't. Your skin has pixels which are (255,255,255,0) aka transparent white. which is showing up in interpolation.

**Comment by peppy** [comment]
Pointless to keep opening these issues, please leave https://github.com/ppy/osu/issues/10715 open to track and don't make any more.

**Comment by enoslayd** [comment]
Sorry, i didnt know it was related to the followpoints at all (and the issue was renamed so....) since this two form different places.
